docs/architecture/PROCESS.md

```diff
@@ -0,0 +1,58 @@
+# ADR Creation Process
+
+1. Copy the `adr-template.md` file. Use the following filename pattern: `adr-next_number-title.md`
+1. Create a draft Pull Request if you want to get an early feedback.
+1. Make sure the context and a solution is clear and well documented.
+1. Add an entry to a list in the [README](./README.md) file.
+1. Create a Pull Request to propose a new ADR.
```

Comment:
Can we just use 1, 2, 3, etc.

Reply:
OK, I'm used to let the markdown viewer to do it for me ;)